lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -11,27 +11,53 @@ class Slideshow extends React.Component {
     super(props);
 
     this.state = {
-      activeSlide: 0
+      currentSlide: 0,
+      animationDirection: 'left'
     };
+
+    this.moveBackward = this.moveBackward.bind(this);
+    this.moveForward = this.moveForward.bind(this);
   }
 
-  moveSlide() {
-    //
+  moveBackward() {
+    const slide = this.state.currentSlide - 1;
+
+    this.setState({
+      currentSlide: slide,
+      animationDirection: 'right'
+    });
+  }
+
+  moveForward() {
+    const slide = this.state.currentSlide + 1;
+
+    this.setState({
+      currentSlide: slide,
+      animationDirection: 'left'
+    });
   }
 
   render() {
     const user = this.props.user;
     const campaign = this.props.campaign;
 
     const slides = this.props.slides.map((slide, index) => {
-      return React.createElement(slide, {key: index, campaign: campaign, user: user});
+      const isActive = this.state.currentSlide == index;
+      const displayClasses = ['slideshow__slide'];
```

Comment:
I think it could be an indicator that the slide content should be separated from the common "slide" aspects shared between them, like `<Slide><InstructionContent /></Slide>` and `<Slide><ContextContent /></Slide>`. This is definitely something that can be broken into another PR though, if you want.